src/effects/VST/VSTEffect.cpp

```diff
@@ -1411,6 +1411,8 @@ bool VSTEffect::ValidateUI(EffectSettings &settings)
    if (GetType() == EffectTypeGenerate)
       settings.extra.SetDuration(mDuration->GetValue());
 
+   FetchSettings(GetSettings(settings));
```

Comment:
The commit that inserts calls to `FetchSettings` should also do it in `LoadFactoryPreset`

Reply:
Good catch, now done